medusa/indexers/tmdb/tmdb.py

```diff
@@ -187,6 +187,16 @@ def search(self, series):
 
         return OrderedDict({'series': mapped_results})['series']
 
+    def _get_shows_countries(self, tmdb_id):
+        """Retrieve show's countries from TMDB
+
+        :param tmdb_id: The shows tmdb id
+        :return: A string with the show's countries
+        """
+        show_info = self._get_show_by_id(tmdb_id)['series']
```

Comment:
What if the tmdb_id used, is incorrect. The id can come from a different indexer, so there is no guarantee it's correct.